ui/src/pages/settings/auto-imports/auto-import.tsx

```diff
@@ -0,0 +1,137 @@
+import { BreadcrumbNav, Button, Checkbox, ColoredBox, Panel, Toolbar } from '@mergestat/blocks'
+import { GithubIcon } from '@mergestat/icons'
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import { Fragment } from 'react'
+
+import SettingsView from 'src/views/settings'
+
+const AutoImports: NextPage = () => {
+  const router = useRouter()
+  const crumbs = [
+    {
+      text: 'Auto imports',
+      onClick: () => router.push('/settings/auto-imports'),
+    },
+    {
+      text: 'MergeStat',
+      startIcon: (
+        <ColoredBox size='10'>
+          <GithubIcon className='t-icon' />
+        </ColoredBox>
+      ),
+    },
+  ]
+
+  const syncsTypesArray = [
+    {
+      type: 'GIT_COMMITS',
+      description: 'Retrieves the commit history of a repo',
+      shortName: 'Git Commits',
+    },
+    {
+      type: 'GIT_COMMIT_STATS',
+      description: 'Retrieves commit stats for a repo',
+      shortName: 'Git Commit Stats',
+    },
+    {
+      type: 'GIT_FILES',
+      description: 'Retrieves files (content and paths) of a git repo',
+      shortName: 'Git Files',
+    },
+    {
+      type: 'GITHUB_PR_COMMITS',
+      description: 'Retrieves commits for all pull requests in a GitHub repo',
+      shortName: 'GitHub PR Commits',
+    },
+    {
+      type: 'GITHUB_PR_REVIEWS',
+      description:
+        'Retrieves the reviews of all pull requests in a GitHub repo',
+      shortName: 'GitHub PR Reviews',
+    },
+    {
+      type: 'GITHUB_REPO_ISSUES',
+      description: 'Retrieves all the issues of a GitHub repo',
+      shortName: 'GitHub Repo Issues',
+    },
+    {
+      type: 'GITHUB_REPO_METADATA',
+      description: 'Retrieves metadata about a GitHub repo',
+      shortName: 'GitHub Repo Metadata',
+    },
+    {
+      type: 'GITHUB_REPO_PRS',
+      description: 'Retrieves all the pull requests of a GitHub repo',
+      shortName: 'GitHub Repo Pull Requests',
+    },
+    {
+      type: 'GITHUB_REPO_STARS',
+      description: 'Retrieves all stargazers of a GitHub repo',
+      shortName: 'GitHub Repo Stars',
+    },
+    {
+      type: 'GIT_REFS',
+      description: 'Retrieves all the refs of a git repo',
+      shortName: 'Git Refs',
+    },
+  ]
```

Comment:
Not sure how this is used, but this list should be retrieved dynamically from the server vs hardcoded in here I believe. @german-mergestat

Reply:
ohh! this is an old file, seems `rebase` brought this through again, but no worries, it is being retrieved from database yep. So, I removed this folder `ui/src/pages/settings/auto-imports`